tests/sync/test_context_manager.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from playwright.sync_api import BrowserType
+
+
+def test_context_managers(browser_type: BrowserType, launch_arguments):
+    with browser_type.launch(**launch_arguments) as browser:
+        with browser.new_context() as context:
+            with context.new_page() as page:
+                page.goto("http://whatsmyuseragent.org/")
```

Comment:
We don't want to rely on external websites for the tests. A simple page.evaluate("1 + 1") would be enough.

Reply:
Fixed